python/tests/unit/utils/data_splitters_test.py

```diff
@@ -39,27 +40,32 @@ def _run_splitter_distributed(
     process_num: int,
     world_size: int,
     init_method: str,
-    edges: list[torch.Tensor],
+    tensors: list[torch.Tensor],
     expected: list[tuple[torch.Tensor, torch.Tensor, torch.Tensor]],
+    is_node_split: bool,
```

Comment:
More readable design + there may be more splitters in the future.
```suggestion
    splitter: Union[HashedNodeSplitter, HashedNodeAnchorLinkSplitter]
```